packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx

```diff
@@ -10,6 +9,10 @@ Installation and usage of our React Native SDK is simple and involves the follow
 First things first, make sure you have set up the development environment for React Native.
 You can find the official guide [here](https://reactnative.dev/docs/environment-setup).
 
+### Expo
+
+Stream's Video SDK is available on Expo through development builds. See the instructions found [here](../expo).
```

Comment:
Since we clearly show a new page and it is evident in the page contents, we can remove it.